config.py

```diff
@@ -8,9 +8,29 @@
 
 
 SOLR_URLS=[
-  'http://ads1/solr/update',
-  'http://ads2/solr/update',
+  'http://localhost:9983/solr/BumblebeeETL/update' 
 ]
+#  'http://ads1/solr/update',
+#  'http://ads2/solr/update',
+
+
+
+SQL = {
```

Comment:
this is a small point, but worth considering: use 'dialect://user:pass@host/db' format - sqlalchemy can parse it